src/styles/home.scss

```diff
@@ -1,11 +1,25 @@
-
 #home {
   text-align: center;
   width: 100%;
+  color: $mainTextColor;
```

Comment:
I wouldn't change things at this point but it's probably good to get in the habit of putting CSS rules in some kind of order every time you write them. We do alphabetical here which is usually the easiest. Of course, Prettier does it for us on save :)

Reply:
I am looking to work on that when I clean up my style files.